### PR TITLE
Stub Methods for Event

### PR DESCRIPTION
**Pull request by Jamie-Unosquare**
Left out DependencyInjection, WebMappingProfile and ServiceMapperProfile so as to not conflict with Campbell's change

### CODE REVIEW

File: v2/AdminCore.Common/Interfaces/IEventService.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+using AdminCore.DTOs.Employee;
+using AdminCore.DTOs.Event;
+using System;
+using System.Collections.Generic;
+
+namespace AdminCore.Common.Interfaces
+{
+  public interface IEventService
+  {
+    IList<EventDto> FindAnnualLeaveByEmployee(int employeeId);
```
Comment (eoinmcafee) [inline v2/AdminCore.Common/Interfaces/IEventService.cs]: Again use Get instead of find.

Comment (eoinmcafee) [reply v2/AdminCore.Common/Interfaces/IEventService.cs]: GetAnnualLeaveByEmployeeId

File: v2/AdminCore.Services/EventService.cs [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    {
+      throw new System.NotImplementedException();
+    }
+
+    public IList<EventDto> MapEventsToDto(List<Event> events)
+    {
+      throw new System.NotImplementedException();
+    }
+
+    public IList<EventDto> MapToEventMessage(String message, Event events, Employee employee, int eventMessageTypeId)
```
Comment (BrownMark) [inline v2/AdminCore.Services/EventService.cs]: Don't need this

**Review by BrownMark — CHANGES_REQUESTED** [review]

File: v2/AdminCore.WebApp/Controllers/Holiday/HolidayController.cs [hunk 3]
```diff
@@ -1,98 +1,99 @@
...
   [ApiController]
-  public class EventController : ControllerBase
+  public class HolidayController : ControllerBase
   {
     private readonly IMapper _mapper;
 
     private readonly IEventService _eventService;
 
-    public EventController(IEventService eventService, IMapper mapper)
+    public HolidayController(IEventService holidayService, IMapper mapper)
```
Comment (BrownMark) [inline v2/AdminCore.WebApp/Controllers/Holiday/HolidayController.cs:29]: It's still the event service

**Review by BrownMark — CHANGES_REQUESTED** [review]

File: v2/AdminCore.WebApp/Controllers/Holiday/HolidayController.cs (hunk 3 above)
Comment (Jamie-Unosquare) [reply v2/AdminCore.WebApp/Controllers/Holiday/HolidayController.cs:29]: So the Interface also needs to be IHolidayService?

File: v2/AdminCore.WebApp/Controllers/Holiday/HolidayController.cs [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+  [Authorize]
+  [Route("[controller]")]
+  [ApiController]
+  public class HolidayController : ControllerBase
+  {
+    private readonly IMapper _mapper;
+
+    private readonly IEventService _eventService;
+
+    public HolidayController(IEventService holidayService, IMapper mapper)
```
Comment (eoinmcafee) [inline v2/AdminCore.WebApp/Controllers/Holiday/HolidayController.cs:29]: rename to eventService

File: v2/AdminCore.WebApp/Controllers/Holiday/HolidayController.cs [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+      throw new System.NotImplementedException();
+    }
+
+    [HttpGet("/{holidayId}")]
+    public EventViewModel GetHolidayByHolidayId(int holidayId)
+    {
+      throw new System.NotImplementedException();
+    }
+
+    [HttpGet("/findByEmployeeId/{employeeId}}")]
```
Comment (eoinmcafee) [inline v2/AdminCore.WebApp/Controllers/Holiday/HolidayController.cs:47]: Need to up do the routes to get instead of find

Comment (Jamie-Unosquare) [reply v2/AdminCore.WebApp/Controllers/Holiday/HolidayController.cs:47]: Asked Mark this yesterday and he suggested keeping the routes as they are so we don't break front end

**Review by BrownMark — APPROVED** [review]

**Pull request merged**